docs/learn-launch.md

```diff
@@ -38,44 +35,38 @@ election. This election will transition the network from PoA into its second pha
 [nominators](maintain-nominator) and the validators themselves.
 
 Time-wise, the transition to Proof-of-Stake will largely depend on when a sufficient number of
-validators have registered and are ready to take over the security of the network. This number can
-be as low as 50 but probably closer to 100, as it was with [Kusama](https://kusama.network).
+validators are ready to take over the security of the network. This transition will probably require
+at least 100 well back validators.
 
-## Sudo to Governance
+## Governance
 
-After the chain has been running well with a sufficiently large validator set, the
-[Sudo](https://youtu.be/InekMjJpVdo) key will enable the suite of governance modules in Polkadot;
-namely, the modules to enable a [Council](learn-governance#council), a
+After the chain has been running well with a sufficiently large validator set, the Sudo key will
+issue a runtime upgrade that enables the suite of governance modules in Polkadot; namely, the
+modules to enable a [Council](learn-governance#council), a
 [Technical Committee](learn-governance#technical-committee), and
 [public referenda](learn-governance#public-referenda). Once a Council and governance tools are in
 place, the public has the avenues to effect changes in the system.
 
+## Removal of Sudo
+
+A runtime upgrade &mdash; either via governance processes or from Sudo itself &mdash; will remove
+the Sudo module from the runtime.
+
 From this point, the network is entirely in the hands of the token holders and no longer under
 control of any centralized authority. Without Sudo, the chain will no longer be a candidate (CC) and
 will be a live, decentralized network.
 
-This period will last approximately three months: ~28 days to table the proposal and turn it into a
```

Comment:
Because we don't know. It could go faster by sudo just making the upgrade to remove itself (almost instantaneous), faster by fast tracking, or it might sit in this period for a long time before the proposal gets made.